lib/proxy_api/tftp.rb

```diff
@@ -1,27 +1,28 @@
 module ProxyAPI
   class TFTP < ProxyAPI::Resource
     def initialize(args)
-      @url     = args[:url] + "/tftp"
-      @variant = args[:variant]
```

Comment:
I can't do that, I'd need to make new instance per individual `kind`. That does not play well with orchestration code and validation mechanisms.